README.md

```diff
@@ -0,0 +1,16 @@
+# audio
```

Comment:
definitly need to improve the read.me
D'ailleurs je serai partant de se faire une petite déterre pour taffer sur la visue web du github (qu'on peut transformer en siteweb ahah)